files/semantic-ui.daterangepicker/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "semantic-ui.daterangepicker",
+  "repo": "BreadMaker/semantic-ui-daterangepicker",
+  "files": {
+    "include": ["daterangepicker.css", "daterangepicker.js", "daterangepicker.min.css", "daterangepicker.min.js"]
```

Comment:
1.3.21 is the latest version according to this https://github.com/BreadMaker/semantic-ui-daterangepicker/releases